types/npm-packlist/index.d.ts

```diff
@@ -0,0 +1,16 @@
+// Type definitions for npm-packlist 1.1
+// Project: https://www.npmjs.com/package/npm-packlist
```

Comment:
It's better to use repository url <https://github.com/npm/npm-packlist> rather than npm package url

Reply:
Good catch, fixed. Though that's what `dts-gen` generated.